src/set_theory/ordinal_topology.lean

```diff
@@ -0,0 +1,252 @@
+/-
+Copyright (c) 2022 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+import set_theory.ordinal_arithmetic
+import topology.algebra.order.basic
+
+/-!
+### Topology of ordinals
+
+We prove some miscellaneous results involving the order topology of ordinals.
+
+### Main results
+
+* `is_closed_iff_sup` / `is_closed_iff_bsup`: A set of ordinals is closed iff it's closed under
+  suprema.
+* `is_normal_iff_strict_mono_and_continuous`: A characterization of normal ordinal functions.
+* `enum_ord_is_normal_iff_is_closed`: The function enumerating the ordinals of a set is normal iff
```

Comment:
Please add namespaces to these names, so that they will be turned into hyperlinks in the docs.